### PR TITLE
Release version v0.9.0

### PR DESCRIPTION
**Pull request by michael-valdron**
### What does this PR do?:

Release changes for AI Software Templates v0.9.0.

### Which issue(s) this PR fixes:

https://issues.redhat.com/browse/DEVAI-185

### PR acceptance criteria:
Testing and documentation do not need to be complete in order for this PR to be approved. We just need to ensure tracking issues are opened and linked to this PR, if they are not in the PR scope due to various constraints.

- [ ] Tested and Verified

  

- [X] Documentation (READMEs, Product Docs, Blogs, Education Modules, etc.)

### CODE REVIEW

**Review by thepetk — APPROVED** [review]
lgtm. Similar to what I mentioned here: https://github.com/redhat-ai-dev/ai-lab-samples/pull/14#pullrequestreview-2405334534, should we add this PR or file as template for future releases?

**Review by Jdubrick — APPROVED** [review]
lgtm

File: RELEASE-NOTES.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Release Notes
```
Comment (maysunfaisal) [inline RELEASE-NOTES.md:1]: same ques as https://github.com/redhat-ai-dev/ai-lab-samples/pull/14/files#r1823032703

Comment (michael-valdron) [reply RELEASE-NOTES.md:1]: @maysunfaisal Answered with https://github.com/redhat-ai-dev/ai-lab-samples/pull/14#discussion_r1823052094 and https://github.com/redhat-ai-dev/ai-lab-samples/pull/14#discussion_r1823057354.

**Review by maysunfaisal — APPROVED** [review]

**Pull request merged**